src/app/components/bitcoin-fees-list/components/fees-list-item.tsx

```diff
@@ -1,4 +1,4 @@
-import { Box, transition } from '@stacks/ui';
+import { Box } from '@stacks/ui';
```

Comment:
If we update this `Box` to be from `leather-styles` maybe we don't need to use `token` anymore?

Reply:
Yep, true, I can go back and do. Was just doing some changes quickly when not on my list of files in the issue. I felt like the PR was getting really big.